engine/crates/composition/Cargo.toml

```diff
@@ -1,10 +1,14 @@
 [package]
-name = "grafbase-composition"
-version = "0.1.0"
+name = "graphql-composition"
```

Comment:
Yes, this is implementing the spec so the idea with the crates.io release is that it will end up making sense as open source that other projects can use.